### PR TITLE
add work dir! fix inconsistent problems when "gop run ." an "gop build ."

### PR DESCRIPTION
**Pull request by sunqirui1987**
add work dir! fix inconsistent problems when "gop run ." an "gop build."

### CODE REVIEW

File: fs/fsutil/dir.go [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+func GetWorkDir() string {
+	dir := getCurrentAbPathByExecutable()
+	if strings.Contains(dir, os.TempDir()) {
+		dir, _ = os.Getwd()
+	}
+	return dir
+}
+
+// 获取当前执行程序所在的绝对路径
```
Comment (xushiwei) [inline fs/fsutil/dir.go:18]: don't use chinese.

**Comment by xushiwei** [comment]
https://github.com/goplus/spx/pull/141 - use this pr to solve the problem.

**Comment by xushiwei** [comment]
is this issue to resolve the problem of clicking app to run in macOS?

**Comment by sunqirui1987** [comment]
> #141

yes

**Pull request merged**